src/main/scala/jtag/JtagStateMachine.scala

```diff
@@ -81,6 +80,7 @@ class JtagStateMachine extends Module() {
   
   withReset (io.asyncReset) {
     val nextState = Wire(JtagState.State.chiselType())
+    nextState := DontCare
```

Comment:
I see a lot of DontCare assigns - how many of these are actually supposed to be DontCare at the top-level, vs. the compiler not catching that it is actually being exhaustively assigned, or assigning DontCare in the cases where it is expected?

Reply:
I'd like to clean it up, in fact I tried to get rid of these or handle them as you say, but it was turning into a time sync. But I agree, they look awkward

Reply:
Probably should at least add a TODO there so we remember to clean them up, maybe also a GitHub issue.

Reply:
Added TODOs for these